orchestra/static/dist/main.js

```diff
@@ -47483,16 +47483,10 @@ function helpers() {
       });
     },
 
-    getAssigmentsOrderedByList: function getAssigmentsOrderedByList(orderedListOfSlugs, assignmentDataList) {
-      return orderedListOfSlugs.reduce(function (acc, slug) {
-        var found = assignmentDataList.filter(function (a) {
-          return a.stepSlug === slug;
-        });
-        if (found.length > 0) {
-          acc.push(found[0]);
-        }
-        return acc;
-      }, []);
+    orderAssigmentsUsingPattern: function orderAssigmentsUsingPattern(orderedListOfSlugs, assignmentDataList) {
```

Comment:
How about `orderAssigmentsUsingListOfSlugs` instead of `orderAssigmentsUsingPattern`? I feel its unclear what pattern means here!

Reply:
@adbharadwaj updated